src/modules/local_position_estimator/BlockLocalPositionEstimator.hpp

```diff
@@ -369,20 +373,20 @@ class BlockLocalPositionEstimator : public control::SuperBlock
 	bool _lastArmedState;
 
 	// masks
-	uint8_t _sensorTimeout;
-	uint8_t _sensorFault;
+	uint16_t _sensorTimeout;
```

Comment:
You need to initialize with 65535 in the constructor.

Reply:
You are right. Sorry

Reply:
@ndepal fixed